deepfence_server/pkg/integration/integration.go

```diff
@@ -55,6 +55,16 @@ func GetIntegration(ctx context.Context, integrationType string, b []byte) (Inte
 	}
 }
 
+func HumanReadableDateFlag(integrationType string) bool {
```

Comment:
Can you rename this to `IsMessagingFormat` or something like that?
This is required for other places too

Reply:
Done.